variables.tf

```diff
@@ -1503,4 +1503,10 @@ variable "add_kms_block_storage_s2s" {
   default     = true
 }
 
+variable "create_s2s_authorization_policy" {
+  description = "Set it to true to create the authorization policy."
+  type        = bool
+  default     = true
+}
+
```

Comment:
I'm wondering if maybe we take this opportunity to rename the variable `add_kms_block_storage_s2s` to `skip_kms_block_storage_s2s_auth_policy` and default it to `false`. And then we name the new variable to `skip_all_s2s_auth_policies`. That way the consumers can choose to skip all, or just to skip the kms_block_storage one (since this one can't be scoped to an instance or resource group).

Any thoughts on this @toddgiguere ?

Reply:
@ocofaigh I think I agree to use "skip" with default "false", as I think most of the time you will want these policies, and "skipping" by setting a variable makes more sense.

Also agree that the variables should end in word "policy" or "policies" to make it more clear what these are for.